ts/components/InstabugAssistanceComponent.tsx

```diff
@@ -0,0 +1,126 @@
+import { none, Option, some } from "fp-ts/lib/Option";
+import I18n from "i18n-js";
+import { BugReporting } from "instabug-reactnative";
+import { H3, Text, View } from "native-base";
+import * as React from "react";
+import { Platform } from "react-native";
+import { connect } from "react-redux";
+import {
+  instabugReportClosed,
+  instabugReportOpened
+} from "../store/actions/debug";
+import { updateInstabugUnreadMessages } from "../store/actions/instabug";
+import { Dispatch } from "../store/actions/types";
+import { instabugMessageStateSelector } from "../store/reducers/instabug/instabugUnreadMessages";
+import { GlobalState } from "../store/reducers/types";
+import ButtonWithImage from "./ButtonWithImage";
+
+type OwnProps = Readonly<{
+  hideComponent: () => void;
```

Comment:
```suggestion
  onReportOpening: () => void;
```
Could it be a better name?
BTW we can add another prop `onReportOpened` fired when we open the Instabug report (if we will adopt this solution based on a timer)

Reply:
there is an existing fired action, `instabugReportOpened` so I opted for `onInstabugReportOpening`. updated within https://github.com/pagopa/io-app/pull/1629/commits/da1bbdfd04c40079d66e00455e3705a3b9a5b0c9